frontend/packages/shared/src/utils/featureToggleUtils.test.ts

```diff
@@ -61,10 +61,8 @@ describe('featureToggle url', () => {
     );
     expect(shouldDisplayFeature('componentConfigBeta')).toBeFalsy();
     expect(shouldDisplayFeature('shouldOverrideAppLibCheck')).toBeTruthy();
-    expect(shouldDisplayFeature('displayPaymentTaskProcessEditor')).toBeTruthy();
```

Comment:
Think we should adapt the url in this test as well?
And maybe add another featureFlag to ensure that all flags are added to the session storage when the persist flag is set? 🫣